src/qubx/trackers/sizers.py

```diff
@@ -0,0 +1,50 @@
+import numpy as np
+
+from qubx import logger
+from qubx.core.basics import Position, Signal
+from qubx.core.strategy import IPositionSizer, StrategyContext
+
+
+class FixedSizer(IPositionSizer):
+    """
+    Simplest fixed sizer class. It uses same fixed size for all signals.
+    We use it for quick backtesting of generated signals in most cases.
+    """
+
+    def __init__(self, fixed_size):
+        self.fixed_size = abs(fixed_size)
+
+    def get_position_size(self, ctx: StrategyContext, signal: Signal) -> float:
+        return signal.signal * self.fixed_size
+
+
+class FixedRiskSizer(IPositionSizer):
+    def __init__(self, max_cap_in_risk: float, max_allowed_position=np.inf):
+        """
+        Create fixed risk sizer calculator instance.
+        :param max_cap_in_risk: maximal risked capital (in percentage)
+        :param max_allowed_position: limitation for max position size
+        """
+        self.max_cap_in_risk = max_cap_in_risk
+        self.max_allowed_position = max_allowed_position
+
+    def get_position_size(self, ctx: StrategyContext, signal: Signal) -> float:
```

Comment:
What is signal doesn't contain the stop information, but instead it's the tracker that sets stop. Will the tracker modify the signal object before calling the position sizer?

Reply:
Yep I think we can add checking this in tracker - if signal has stop level - do not change it, otherwise set it's own